src/content/docs/en/guides/integrations-guide/solid-js.mdx

```diff
@@ -83,16 +83,30 @@ Most package managers will install associated peer dependencies as well. If you
 
 Then, apply the integration to your `astro.config.*` file using the `integrations` property:
 
-```js title="astro.config.mjs" ins={2} ins="solid()"
+```js title="astro.config.mjs" ins={2} ins="solidJs()"
```

Comment:
Just confirming for myself that `astro add solid` does add the solid integration named this way, so seems like a change consistent with the CLI tool!

Reply:
Yes, that's correct. I'm trying to respect the installation method through CLI as much as possible.